docs/getting_started.md

```diff
@@ -0,0 +1,140 @@
+# Getting started with decidim
+
+## What is and what isn't decidim?
+
+Decidim is a set of libraries to create a participatory democracy framework on top of a Ruby on Rails app. This system allows having the system code separated from custom code for each installation and still enabling easy updates.
+
+These libraries are published to Rubygems.org, so you can add decidim to your Ruby on Rails app as external dependencies.
+
+If you want to start your own installation of decidim, you don't need to clone this repo. Keep reading to find out how to install decidim.
+
+## Creating your Decidim app
+
+### Using docker [experimental]
+
+> *Please note that this is **experimental***
+
+Make sure you have docker v17 at least. `cd` to your preferred folder and run this command:
+
+```
+docker run --rm -v $(pwd):/tmp codegram/decidim bash -c "bundle exec decidim /tmp/decidim_application"
+```
+
+This will create a `decidim_application` Ruby on Rails app using decidim in the current folder. It will install the latest released version of the gem.
+
+### Step by step
+
+First of all, you need to install the `decidim` gem:
+
+```
+$ gem install decidim decidim-core
```

Comment:
It doesn't seem necessary anymore, since `decidim-core` is a `decidim` dependency. I observed the same thing when I installed `decidim` a month ago or so.